### ISSUE TITLE
why not a flashWeave R package?

### ISSUE THREAD

**Issue by Gian77** [meringlab/FlashWeave.jl#38]
Just as a curiosity, whey this cool tool has been implemented in Julia? 
Thnaks,
Gian

**Comment by jtackm** [comment]
FlashWeave requires many custom algorithms that need to be both really fast and maintainable. We started out in python, but quickly realized that cython/numba were too much overhead or too unrealiable for our needs. I suspect R (with Rcpp) would have caused similar issues. Julia made development of a package like this much easier: being able to code in just one dynamic language with nice generics while still generating fast code.

Regarding running FlashWeave from R: I've seen people using it with JuliaCall (https://hwborchers.github.io/). Perhaps that would work for you? It also shouldn't be hard to make a proper R package (using this interoperability), but I'm not deep in the R ecosphere, so would best leave such a project to somebody with previous experience.

**Comment by jtackm** [comment]
Yes, I agree it would be great to have FlashWeave also available as native R or Python packages. I'm a bit strained on time currently, but happy to support such efforts if anybody is willing to look into it.

In the meantime, the commands are hopefully simple enough and the language close enough to Python/R such that it shouldn't be too much overhead.
